src/app/components/multiselect/multiselect.ts

```diff
@@ -1412,7 +1412,7 @@ export class MultiSelect implements OnInit, AfterViewInit, AfterContentInit, Aft
     }
 
     searchFields() {
-        return (this.filterBy || this.optionLabel || 'label').split(',');
+        return this.filterBy?.split(',') || this.filterFields || [this.optionLabel];
     }
```

Comment:
I believe this change will break if filterBy,filterFields,and optionLabel is not  defined

I have not tested this but I think the proper change is

`return this.filterBy?.split(',') || this.filterFields || (this.optionLabel?[this.optionLabel]:['label']);`

Reply:
Yes you are correct, the same applies to dropdown too. I will change all it ASAP.